nipype/interfaces/setup.py

```diff
@@ -24,6 +24,7 @@ def configuration(parent_package='', top_path=None):
     config.add_subpackage('mne')
     config.add_subpackage('mrtrix')
     config.add_subpackage('mrtrix3')
+    config.add_subpackage('niftyseg')
```

Comment:
Ok, I don't have an special interest for having only one package. Let's observe that preference (reg/seg/fit packages) :+1: .

So if we keep the 3 packages structure, then I'm suggesting here to import from niftyreg.base and remove the duplicated code from niftyseg and niftyfit.

WDYT?